docs/what-is-nuget.md

```diff
@@ -92,13 +92,14 @@ When you have a choice, we recommend using PackageReference. `packages.config` i
 
 So far you've learned the following characteristics of NuGet:
 
-- NuGet provides the central nuget.org repository with support for private hosting.
+- NuGet provides access to a public package repository, nuget.org, but also supports privately hosted respositories even on the local system.
```

Comment:
I think maybe this could say: 
public, private repositories and local feeds. 

Potentially it sohuld be linked to https://docs.microsoft.com/en-us/nuget/hosting-packages/overview.

Reply:
> NuGet provides access to a central package repository known as nuget.org with support for privately hosted feeds.

Maybe?